adoc/chapters/programming_interface.adoc

```diff
@@ -18019,14 +18019,9 @@ vec<ConvertT, NumElements> convert() const
 a@
 [source]
 ----
-template <typename asT> asT as() const
+template <typename AsT> AsT as() const
 ----
-   a@ Bitwise reinterprets this SYCL [code]#vec# as a SYCL [code]#vec# of a
-      different element type and number of elements specified by [code]#asT#.
-      The new SYCL [code]#vec# type must have the same storage size in bytes as
-      this SYCL [code]#vec#, and the size of the elements in the new SYCL
-      [code]#vec# ([code]#NumElements * sizeof(DataT)#) must be the same as the
-      size of the elements in this SYCL [code]#vec#.
+   a@ Equivalent to [code]#bit_cast<AsT>(*this)#.
```

Comment:
Which `bit_cast` are you talking about?
`std::bit_cast`? If so it should be precised.

Reply:
Good point.  Fixed in 44e1fea7a308e38dc15b274c9a940617fa750c9a.